lib/fx/adapters/postgres/functions.rb

```diff
@@ -10,16 +10,19 @@ class Functions
         # dumpable into `db/schema.rb`.
         FUNCTIONS_WITH_DEFINITIONS_QUERY = <<-EOS.freeze
           SELECT
-              pp.proname AS name,
-              pg_get_functiondef(pp.oid) AS definition
+            pp.proname AS name,
+            pn.nspname AS schema,
+            pg_get_functiondef(pp.oid) AS definition,
+            current_schema() AS current_schema
           FROM pg_proc pp
           JOIN pg_namespace pn
               ON pn.oid = pp.pronamespace
           LEFT JOIN pg_depend pd
               ON pd.objid = pp.oid AND pd.deptype = 'e'
           LEFT JOIN pg_aggregate pa
               ON pa.aggfnoid = pp.oid
-          WHERE pn.nspname = 'public' AND pd.objid IS NULL
+          WHERE pn.nspname NOT IN ('pg_catalog', 'information_schema')
```

Comment:
Some strategies go as far as ignoring everything in `pg_*`. I'm considering making that change still.

(Unfortunately I have not found any other way to isolate user-created functions, and there are many examples that take a similar approach to what I have done here.)

Reply:
@smudge I think you can use [this](https://github.com/teoljungberg/fx/compare/master...joelind:fx:dump-functions-from-all-schemas) (I stupidly branched and created a PR before I saw this one)